microcosm_flask/swagger/definitions.py

```diff
@@ -168,17 +168,27 @@ def query_param(name, field, required=False):
     return swagger.QueryParameterSubSchema(**parameter)
 
 
-def path_param(name, param_type="string"):
+def path_param(name, ns):
     """
     Build a path parameter definition.
 
     """
-    return swagger.PathParameterSubSchema(**{
+    if ns.identifier_type == "uuid":
```

Comment:
At the moment, this will only handle string and uuid arguments well, but that's still a win.